graylog2-web-interface/src/components/content-packs/ContentPackSelection.jsx

```diff
@@ -27,6 +28,15 @@ import { hasAcceptedProtocol } from 'util/URLUtils';
 
 import style from './ContentPackSelection.css';
 
+const HeaderText = styled.span`
+  overflow-wrap: anywhere;
+`;
+
+const HeaderIcon = styled(Icon)`
+  padding-top: 3px;
```

Comment:
We can also try and use `theme.spacings.xs`. 
example:
```suggestion
  padding-top: ${theme.spacings.xs};
```